tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-ci-pipeline.yml

```diff
@@ -38,6 +39,8 @@ jobs:
     condition: succeededOrFailed()
 
     # Entry point for all ORTModule tests
+    # Test run not from inside the /build/RelWithDebInfo/ to ensure the
+    # tests pick up the installed wheel package of onnxruntime
```

Comment:
```suggestion
```

Reply:
Same as above. Updated the comment to indicate the deletion of the onnxruntime folder in the build dir.